docs/api_specs/status_events_specs.md

```diff
@@ -37,6 +37,7 @@ Indicates that an invalid exit is occurring. It should be challenged.
 ```
```

Comment:
@pdobacz this document says `User should exit.`. What it really means is that the watcher should exit with all it's UTXOs? IS that what User means in this case?

Reply:
The User here is the user of the Watcher. Watcher is understood as the app, not a role. 

In other words a "user" here is someone who holds value on the child chain, is interested in keeping that safe and uses the Watcher to that end